edit/workload/index.vue

```diff
@@ -578,30 +578,32 @@ export default {
         }
       }
 
-      const containerResources = template.spec.containers[0].resources;
-      const nvidiaGpuLimit = template.spec.containers[0].resources?.limits[GPU_KEY];
-
-      // Though not required, requests are also set to mirror the ember ui
-      if (nvidiaGpuLimit > 0) {
-        containerResources.requests = containerResources.requests || {};
-        containerResources.requests[GPU_KEY] = nvidiaGpuLimit;
-      }
-
-      if (!this.nvidiaIsValid(nvidiaGpuLimit) ) {
-        try {
-          delete containerResources.requests[GPU_KEY];
-          delete containerResources.limits[GPU_KEY];
+      if (template.spec.containers && template.spec.containers[0]) {
```

Comment:
This could just be `template.spec.containers?.[0]`, but it achieves the same result